composer/algorithms/low_precision_groupnorm/low_precision_groupnorm.py

```diff
@@ -52,8 +52,10 @@ class LowPrecisionGroupNorm(Algorithm):
 
     def __init__(self, apply_at: Event = Event.INIT):
         self.apply_at = apply_at
-        if self.apply_at not in {Event.INIT, Event.AFTER_LOAD}:
-            raise ValueError('LowPrecisionGroupNorm only supports application on Event.INIT and Event.AFTER_LOAD.')
+        if self.apply_at not in {Event.INIT, Event.BEFORE_LOAD, Event.AFTER_LOAD}:
+            raise ValueError(
+                'LowPrecisionGroupNorm only supports application on Event.INIT, Event.BEFORE_LOAD, and Event.AFTER_LOAD.'
+            )
```

Comment:
Why? I think it should stay at INIT/AFTER_LOAD?

Reply:
I could choose to do model surgery at BEFORE_LOAD and it would have the same effect as at INIT right? So we do support application at BEFORE_LOAD in addition to the other two Events.